package.json

```diff
@@ -16,6 +16,13 @@
                 "peerDependencies": {
                     "mocha-junit-reporter": "*"
                 }
+            },
+            "@kubernetes/client-node": {
+                "dependencies": {
+                    "@types/node": "^10.12.0",
```

Comment:
Where are these versions coming from?

Reply:
I so wish it was easier to leave comments in package.json files. I'll see if I can inject a comment field.

TL;DR is that these come from the `devDependencies` of `@kubernetes/client-node` and the are needed for the `prepare` lifecycle hook. Surprisingly the `prepare` step of that npm package runs tsc and makes the `dist` folder and for it to pass the `@types/node` must be an older version.

Reply:
added
```
                "//1": "these deps are from devDependencies of @kubernetes/client-node and needed for 'prepare' lifecycle hook",
                "//2": "https://github.com/kubernetes-client/javascript/blob/763decf6e09d0e246b80cefdb2c5fd570373801a/package.json#L72",
```